plugins/synaptics-rmi/fu-synaptics-rmi-device.c

```diff
@@ -1007,6 +1007,9 @@ fu_synaptics_rmi_device_init (FuSynapticsRmiDevice *self)
 	fu_device_set_name (FU_DEVICE (self), "Touchpad");
 	fu_device_set_remove_delay (FU_DEVICE (self), FU_DEVICE_REMOVE_DELAY_RE_ENUMERATE);
 	fu_device_set_version_format (FU_DEVICE (self), FWUPD_VERSION_FORMAT_TRIPLET);
+	/* there are a lot of Synaptics HID peripherals, but not all respond
+	 * well to probing -- so limit to ones with issued updates */
+	fu_device_add_flag (FU_DEVICE (self), FWUPD_DEVICE_FLAG_ONLY_SUPPORTED);
```

Comment:
I don't know the only supported flag makes sense.  We have a bunch of GUIDs exported by the devices that allow the same payload to span across multiple systems, don't we?  So if the "wrong" GUID is used then this wouldn't work on newer fwupd versions but it would on older.

Reply:
>  We have a bunch of GUIDs exported by the devices that allow the same payload to span across multiple systems, don't we

I don't think so; I think it's only Lenovo using this in anger.

Reply:
But the point being why even export those GUIDs if they effectively can't be used.  Unless we find out this is widespread, I would rather a blacklist solution via a quirk.